VERSION

```diff
@@ -1 +1 @@
-1.7.0
+1.7.1
```

Comment:
Please do not change the version inside the PR. While in this case it will probably go out just after merge, such practice would create problems for us very easily (imagine multiple PRs). It is better to leave the version management up to maintainers.